apps/kuikkadb/lib/kuikkadb/role_permissions.ex

```diff
@@ -36,4 +36,19 @@ defmodule KuikkaDB.RolePermissions do
   """
   @spec delete(Keyword.t) :: {:ok | :error, term}
   defdelete delete(conds), table: :role_permissions
+
+  @spec get_permission_with_role(binary, binary) :: {:ok |:error, term}
+  defquery get_permission_with_role(role_id, permission_id) do
```

Comment:
Remove this query and just modify `get_with_role` function to include permissions in `KuikkaDB.Users`